tests/test_result_to_dict.py

```diff
@@ -1,4 +1,5 @@
 import enum

```

Comment:
question. 
at the result_to_dict documentation we have a list of supported types. 
Do we have tests for all the types that we say that we are supporting?

Reply:
I have added all the types to the document according to the tests